x/ccv/provider/types/proposal.go

```diff
@@ -92,6 +95,21 @@ func (cccp *ConsumerAdditionProposal) ProposalType() string {
 	return ProposalTypeConsumerAddition
 }
 
+// ValidatePSSFeatures returns an error if the `topN` and `validatorsPowerCap` parameters are no in the correct ranges
+func ValidatePSSFeatures(topN uint32, validatorsPowerCap uint32, error *errorsmod.Error) error {
```

Comment:
nits: wrapping the returned error of `ValidatePSSFeatures` is cleaner than passing it as an argument.

Reply:
Yes, that's much cleaner. Thanks for the suggestion.